UIKitDz/UIKitDz/PaymentViewController.swift

```diff
@@ -0,0 +1,128 @@
+//
+//  PaymentViewController.swift
+//  UIKitDz
+//
+//  Created by angelina on 29.09.2022.
+//
+
+import UIKit
+
+/// Контроллер оплаты
+final class PaymentViewController: UIViewController {
+
+// MARK: - Private properties
+    private lazy var orderLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Ваш заказ:"
+        label.textAlignment = .left
+        label.frame = CGRect(x: 40, y: 150, width: 200, height: 30)
+        label.font = label.font.withSize(22)
+        return label
+    }()
+
+    private lazy var pizzaLabel: UILabel = {
+        let label = UILabel()
+        label.text = ""
+        label.textAlignment = .left
+        label.numberOfLines = 0
+        label.frame = CGRect(x: 40, y: 180, width: 300, height: 180)
+        label.font = label.font.withSize(25)
+        return label
+    }()
+
+    private lazy var payByCardLabel = createLabel(title: "Оплата картой", coordinateY: 600)
+    private lazy var payByCashLabel = createLabel(title: "Наличными", coordinateY: 640)
+    private lazy var cardSwitch = createSwitch(coordinateY: 600)
+    private lazy var cashSwitch = createSwitch(coordinateY: 640)
+
+    private lazy var payButton: UIButton = {
+        let button = UIButton(configuration: UIButton.Configuration.filled())
+        button.tintColor = .orange
+        button.setTitle("\u{F8FF} Pay", for: .normal)
+        button.addTarget(self, action: #selector(payAction(_:)), for: .touchUpInside)
+        button.frame = CGRect(x: 50, y: 700, width: 300, height: 50)
+        return button
+    }()
+
+    private lazy var reviewButton: UIButton = {
+        let button = UIButton(configuration: UIButton.Configuration.filled())
+        button.tintColor = .black
+        button.setTitle("Оставить отзыв", for: .normal)
+        button.addTarget(self, action: #selector(reviewAction(_:)), for: .touchUpInside)
+        button.frame = CGRect(x: 50, y: 760, width: 300, height: 50)
+        return button
+    }()
+
+    weak var delegate: DelegateViewController?
+    var orderIngredients: [String] = []
+    var pizzaIngredients: PizzaIngredients?
+    var pizzaName = ""
+
+// MARK: - Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        view.backgroundColor = .white
+        setupUI()
+    }
+
+// MARK: - Private methods
+    private func setupUI() {
+        navigationItem.title = "Оплата"
+        pizzaLabel.text = "1 \(pizzaName)\n" + (pizzaIngredients?.getPizzaIngredients())!
```

Comment:
```suggestion
        pizzaLabel.text = "1 \(pizzaName)\n" + (pizzaIngredients?.getPizzaIngredients())!
```
убрать ! и заменить  +  на интерполяцию